### ISSUE TITLE
НОВЫЙ МАРКЕТ БОТ

### ISSUE THREAD

**Issue by DotkaProDotka** [Maxx53/SCMBot#596]
Всем Добрый День , подскажите есть ли уже какой либо рабочий новый маркет бот на выставление запросов на покупку и автопродажу ? Подскажите если не сложно !!! Буду Очень благодарен !!!

**Comment by Igzibed** [comment]
Да, я думаю, все тут занимаются поиском актуального на данный момент бота. Вместо этого лучше было бы предлагать идеи Максу для реконструкции старого.

**Comment by stranger2001** [comment]
@Igzibed согласен, лучше старый переделать под некоторые определенные цели, а то что сейчас творится на ТП никакой прибыли не принесет уже, т.к. скупка плохо идет и запросы на покупку уже в минус давно стоят. Но всё же помимо этого есть способы как сделать деньги другими способами  на ТП.

**Comment by DotkaProDotka** [comment]
Это какими же ?

**Comment by DotkaProDotka** [comment]
И на счет того что не приносит бабки , я за два дня набил около 150 бачей вручную выставляя ордера ,а имея бота который бы просто выставлял ордера и автоматом перепродавал бы , Можно было бы в разы круче мутить

**Comment by stranger2001** [comment]
@DotkaProDotka за первых 2 дня может быть) и еще смотря какие шмотки) и я к тому, что таких юзеров , которые будут ботоводить будет опять хуева туча и ты просто не сможешь ниче купить, все будут на копейку выше тебя ставить постоянно, в следствии чего уходить будешь в - . Вот ты набил 150 бачей ок., почему ты тоже самое не делал с кейсами раньше например?) такая же система, тоже лоты можно ставить)) наверное потому что цена скупки была всего лишь на 10 копеек ниже цены продажи, и ты бы просто уходил в -, с обычными шмотками будет то же самое)
Например тот же редлайн. поставишь покупку по 380 как сейчас, у тебя купится 8 штук по 370 и одна за 300р, из за тех, что купились за 370 ты уйдешь тупо в - и тот что купился за 300 его не окупит, лучше делать другими способами шо я говорил

**Comment by DotkaProDotka** [comment]
stranger2001 а тут есть вещи которые продаются по 1-5 штук в день , и которые мало юзабельны но их можно урвать на 10-15 баксов дешевле , а не на пару копеек ...

**Comment by DotkaProDotka** [comment]
Знает ли кто-то какую либо программу на перепродажу хотя бы ?

**Comment by stranger2001** [comment]
"ведь теперь все лоты с выше 3 баксов требуют подтверждения и вырубить эту хрень нельзя" , чето это совсем печально:/

**Comment by DotkaProDotka** [comment]
Подтверждения при перепродаже , это не так сложно , просто врубить подтверждение через почту и создать прогу на автоподтверждение продажи

**Comment by stranger2001** [comment]
@DotkaProDotka SCMBOT попробуй) , но я не знаю можно ли там массово продавать или нет.
